.coveragerc

```diff
@@ -1110,6 +1110,7 @@ omit =
     homeassistant/components/seven_segments/image_processing.py
     homeassistant/components/seventeentrack/sensor.py
     homeassistant/components/shelly/binary_sensor.py
+    homeassistant/components/shelly/bluetooth/scanner.py
```

Comment:
Can we add tests to cover this file? I am trying to add tests coverage for Shelly since it is the only part missing to qualify for platinum scale.

Reply:
done